src/view.tsx

```diff
@@ -102,7 +130,11 @@ export class ThemeEditorView extends VDomRenderer<ThemeEditorModel> {
       <FormComponent
         schema={this.model.schema}
         formData={this.model.formData}
+        cssProperties={this.model.cssProperties}
```

Comment:
You should not pass data that are not use directly by a component. You should directly provide the `onSubmit` callback that does the job.

Reply:
I don't how to fix this. There is something I may have not well understood in the way it works.